docs/modules/ROOT/pages/sql_stock_ticker_cloud.adoc

```diff
@@ -325,7 +325,7 @@ SELECT
      pr.average AS Average,
      ROUND(((tro.price/pr.average)-1)*100,2) AS Percent_Change,
      CASE
-          WHEN (ROUND(((tro.price/pr.average)-1)*100,2) > 1) THEN 'Up'
+          WHEN (ROUND(((tro.price/pr.average)-1)*100,2) > 0) THEN 'Up'
```

Comment:
this is spotted by Isaac in the pr https://github.com/hazelcast-guides/sql_stock_ticker_cloud/pull/8/files#diff-001c02a5aee369098d6229eadcd0982512ed2542872ec56d0b114e34584cc87aR328